theme.json

```diff
@@ -597,17 +597,58 @@
 			},
 			"core/quote": {
 				"border": {
-					"color": "var(--wp--preset--color--primary)",
-					"style": "solid",
-					"width": "0 0 0 1px"
+					"radius": "var(--wp--preset--spacing--20)"
+				},
+				"color": {
+					"background": "var(--wp--preset--color--base-2)"
+				},
+				"css": "& :where(p) {margin-block-start:0;margin-block-end:var(--wp--preset--spacing--10);} &.is-style-plain :where(cite) {font-size:var(--wp--preset--font-size--small);}",
```

Comment:
> & :where(p) {margin-block-start:0;

That should probably be resolved in core eventually. Not sure if there's an issue yet.